src/styles/blocks/calendar.scss

```
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap_square;
+  width: ($size_square * 7) + ($gap_square * 6);
+
+  &_day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $size_square;
+    height: $size_square;
+    background-color: #eee;
+    border: 1px solid #000;
+    padding: 10px;
+    transition: 0.5s;
```

you should specify what properties will change on transition:
```suggestion
    transition: transform 0.5s, background-color 0.5s;
```